### DISCUSSION TITLE
Testing forked version

### DISCUSSION THREAD

**Discussion by raylphevan** [coreybutler/nvm-windows#742] [Support] [answered]
Hi everyone,

I recently forked this project to see if I could add something to it and am wondering how I can test what I've added? I've tried following the steps from the README section "build from source" but once I've downloaded it, I just get the message: 
"'nvm' is not recognized as an internal or external command, operable program or batch file." 

Thank you for your time!

**Comment by coreybutler** [comment, marked as answer]
@raylphevan Have you seen these pages?

- [Build Yourself](https://github.com/coreybutler/nvm-windows/wiki/Build-Yourself)
- [Hacking on NVM4W](https://github.com/coreybutler/nvm-windows/wiki/Hacking)

**Reply by raylphevan** [reply]
Hi Corey.

Thanks for sharing those. I tried the steps on the Build Yourself page and ended up with the same result. I decided to remove my changes and clone my forked repo again on the branch which has no changes and went through the hacking page. I tried this step: 
https://github.com/coreybutler/nvm-windows/wiki/Hacking#building-nvm-for-windows

and I was met with these:

go run nvm.go list
go: github.com/coreybutler/go-fsutil@v1.1.1 (replaced by C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil): reading C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil\go.mod: open C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil\go.mod: The system cannot find the path specified.
go: github.com/coreybutler/go-fsutil@v1.1.1 (replaced by C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil): reading C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil\go.mod: open C:\Users\corey\OneDrive\Documents\workspace\oss\coreybutler\go-fsutil\go.mod: The system cannot find the path specified.

**Reply by coreybutler** [reply]
There was a local override in the go.mod file. I've updated this in the master branch (or just comment out line number 5 of go.mod on your local copy). That will allow `go run nvm.go list` to run.

**Reply by raylphevan** [reply]
~~I fetched and merged with the master branch and I was able to get rid of the messages. It did allow it to run and it is running as I was expecting it to with the minor change I made. However, I am still having trouble downloading it onto my local machine. I installed the Inno Setup compiler to help install my version to my system and it does, but when I run the command nvm, I get the same message stating that it is not recognized.~~

EDIT: It worked after following the Build Yourself page steps.

Thank you for helping me figure this out. I appreciate it a lot.
